docs/package.json

```diff
@@ -12,7 +12,7 @@
   "private": true,
   "scripts": {
     "start": "npm run browser:development",
-    "browser:development": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --inline --host=",
+    "browser:development": "webpack-dev-server --config webpack-dev-server.config.js --progress --colors --inline",
```

Comment:
Did you figure out your problem with this? I remember merging the `--host=` argument as it didn't break anything for me.